lib/src/define_env_settings.dart

```diff
@@ -0,0 +1,168 @@
+import 'dart:io';
+
+import 'package:checked_yaml/checked_yaml.dart';
+import 'package:console/console.dart';
+import 'package:define_env/src/env_settings.dart';
+
+EnvSettings? loadEnvSettings(String envSettingsPath) {
```

Comment:
This file is too big.
You could refactor this into multiple files.

I would suggest bringing all these files under

`lib/src/settings`